### PR TITLE
Cut v2.30.1

### PR DESCRIPTION
**Pull request by codesome**
DO NOT SQUASH

### CODE REVIEW

**Comment by codesome** [comment]
DCO fails with `Commit sha: 275ecf8, Author: Nick Pillitteri, Committer: Ganesh Vernekar; Expected "Nick Pillitteri 56quarters@users.noreply.github.com", but got "Nick Pillitteri nick.pillitteri@grafana.com".`. I guess that is not an issue, so I am setting DCO to pass.

**Comment by roidelapluie** [comment]
The security fix is not complete.

We hide it from /metrics but not from /config.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 2.30.1 / 2021-09-27
+
+* [BUGFIX] promtool rules backfill: Prevent creation of data before the start time. #9339
+* [BUGFIX] promtool rules backfill: Do not query after the end time. #9340
+* [BUGFIX] Remote Write: Redact remote write URL when used for metric label. #9383
```
Comment (roidelapluie) [inline CHANGELOG.md:5]: This is an ENHANCEMENT since URL is a <string>, which should not contain secrets.

**Comment by codesome** [comment]
I have rebased the PR and updated the CHANGELOG

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 2.30.1 / 2021-09-28
+
+* [ENHANCEMENT] Remote Write: Redact remote write URL when used for metric label. #9383
+* [ENHANCEMENT] UI: Redact remote write URL in the `/config` page. #9408
```
Comment (roidelapluie) [inline CHANGELOG.md:4]:
```suggestion
* [ENHANCEMENT] UI: Redact remote write URL and proxy passwords in the `/config` page. #9408
```

**Review by roidelapluie — APPROVED** [review]

**Comment by codesome** [comment]
I merged it with squash :facepalm: 
No worries, the `main` branch has the individual commits

**Pull request merged**